client/app/ip/ip/legacyOrder/ip-ip-legacyOrder.controller.js

```diff
@@ -55,6 +57,16 @@ angular
       this.$scope.backToContracts = () => this.backToContracts();
       this.$scope.getResumePrice = price => this.getResumePrice(price);
       this.$scope.confirmOrder = () => this.confirmOrder();
+
+      if (this.isOrderingFromDrp()) {
```

Comment:
```js
  this.$scope.closeModal = () => this.closeModal();
} // $onInit

closeModal(){
  if (this.isOrderingFromDrp()) {
    return this.$state.go('^');
  } 

  this.$scope.resetAction();
}
```

Reply:
As seen together, I'll keep it like this